src/components/forms/DatePicker/constants.ts

```diff
@@ -34,3 +34,6 @@ export const YEAR_CHUNK = 12
 export const DEFAULT_MIN_DATE = '0000-01-01'
 export const DEFAULT_EXTERNAL_DATE_FORMAT = 'MM/DD/YYYY'
```

Comment:
I am good with passing the Happo diffs if you can explain why the switch to all caps for the help text? my thoughts would be to not change the default text if it can be avoided, it was originally in lowercase, which an argument could be made is more legible for folks with certain vision impairments.

Reply:
I don't know why the case changed. I see what you mean, but I don't see anything in my patch that would cause it to do that.